composer.json

```diff
@@ -20,26 +20,27 @@
         }
     },
     "require": {
-        "php": ">=7.2",
-        "league/flysystem": "^2.1|^3.0",
-        "symfony/config": "^4.2|^5.0|^6.0",
+        "php": "^8.0",
+        "league/flysystem": "^3.0",
+        "symfony/config": "^4.4|^5.0|^6.0",
```

Comment:
I think we could drop support for Symfony 4.4, 5.0, 5.1, 5.2 and 5.3 as well. People can easily keep the v2 (no big change) and all these versions are either EOL or son to be EOL. Let's focus on 5.4 + 6.0 IMO.